### ISSUE TITLE
Update API Example in backend to use the new /content/_search endpoint

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19691]
We need to update this screen to show an example using the new /api/content/_search POST method

![Screen Shot 2020-12-10 at 4 54 32 PM](https://user-images.githubusercontent.com/934364/101834754-bf49e400-3b08-11eb-8e06-913b1d1d76a2.png)

The example should display a curl example like:
```
curl -XPOST 'http://localhost:8080/api/content/_search' \
--header 'Content-Type: application/json' \
--data-raw '{
      	 "query": "+structurename:webpagecontent",
       	 "sort":"modDate",
       	 "limit":20,
       	 "offset":0
 }'
```
but with the correct values filled in. Be careful of escaping quotes as well.

Additionally we can remove the link to the xml example and just have the json example link that does the actual post and displays the results, rather than the GET link we have now.

**Comment by john-thomas-dotcms** [comment]
So just to be clear, I think we need to add a new example, not replace the existing one.

The existing example uses a GET, so it's easy to test from any browser, but I think the _search endpoint is a POST, which requires special tools (like postman).  I think it's important that we keep a really easy to use example in there (without the need for a POST).

**Comment by john-thomas-dotcms** [comment]
In addition, it might also be a good time to think about adding a GraphQL example in there as well.  We have alot more customers using this now, so an easy to find example could be really useful for them.

**Comment by alfredo-dotcms** [comment]
![image](https://user-images.githubusercontent.com/37185433/102146387-7195ea00-3e2e-11eb-8d7e-c2209379b550.png)
@wezell do you also want me to remove the XML href in the `history` tab? or just the one here:
https://user-images.githubusercontent.com/934364/101834754-bf49e400-3b08-11eb-8e06-913b1d1d76a2.png

**Comment by alfredo-dotcms** [comment]
PR: https://github.com/dotCMS/core/pull/19705

**Comment by wezell** [comment]
Looking good - a few things
- In the Dialog, let's wrap the code examples in a <code></code> block and use a fixed width font.
- Use the short curl args, e.g. `-H` for `--header` and `-d` for `data-raw`
- Move the REST API Call URLEncoded below the post
- Use the POST for the JSON link
- replace the json for the link with the [API] icon 
![Screen Shot 2020-12-15 at 11 50 58 AM](https://user-images.githubusercontent.com/934364/102245726-e703c800-3ecb-11eb-85a4-9cfbda4e5a45.png)

**Comment by alfredo-dotcms** [comment]
@wezell done:
https://github.com/dotCMS/core/pull/19705/commits/a86f7d62cf23adc7f48cd5c5a2ca467fd2931548

**Comment by alfredo-dotcms** [comment]
Preview
![image](https://user-images.githubusercontent.com/37185433/102270083-2c7fbf80-3ee3-11eb-9695-4fcde9dcf98f.png)

**Comment by nollymar** [comment]
Passed internal qa

**Comment by bryanboza** [comment]
Fixed, tested on release-21.02 // Postgres // FF
